### PR TITLE
Improve NodeJs get payment_methods snippets

### PR DESCRIPTION
**Pull request by lucmantovani**
## Description
Improving NodeJs snippet to get payment_methods. The previous snippet ignored that the method return a Promise, which could cause integration issues.

### Checklist:
Before sending your contribution, please specify the following: 

- [X] This request modifies code snippets. 
- [X] The contribution aligns with the information stated in the repository wiki.

### CODE REVIEW

File: guides/online-payments/checkout-api/v2/other-payment-ways.pt.md [hunk 1]
```diff
@@ -51,7 +51,8 @@ Consulte os meios de pagamento disponíveis sempre que necessite.
 var mercadopago = require('mercadopago');
 mercadopago.configurations.setAccessToken(config.access_token);
 
-payment_methods = mercadopago.get("/v1/payment_methods");
+response = await mercadopago.get("/v1/payment_methods");
```
Comment (elvisjhonataoliveira) [inline guides/online-payments/checkout-api/v2/other-payment-ways.pt.md:54]:
```suggestion
var response = await mercadopago.get("/v1/payment_methods");
```

File: guides/online-payments/checkout-api/v2/other-payment-ways.pt.md [hunk 2]
```diff
@@ -568,7 +569,8 @@ Consulte os meios de pagamento disponíveis sempre que necessite.
...
```
Comment (elvisjhonataoliveira) [inline guides/online-payments/checkout-api/v2/other-payment-ways.pt.md:572]:
```suggestion
var response = await mercadopago.get("/v1/payment_methods");
```

File: guides/online-payments/checkout-api/v2/other-payment-ways.pt.md [hunk 3]
```diff
@@ -1064,7 +1066,8 @@ Consulte os meios de pagamento disponíveis sempre que necessite.
...
```
Comment (elvisjhonataoliveira) [inline guides/online-payments/checkout-api/v2/other-payment-ways.pt.md:1069]:
```suggestion
var response = await mercadopago.get("/v1/payment_methods");
```

File: guides/online-payments/checkout-api/v2/other-payment-ways.pt.md [hunk 4]
```diff
@@ -1520,7 +1523,8 @@ Consulte os meios de pagamento disponíveis sempre que necessite.
...
```
Comment (elvisjhonataoliveira) [inline guides/online-payments/checkout-api/v2/other-payment-ways.pt.md:1526]:
```suggestion
var response = await mercadopago.get("/v1/payment_methods");
```

File: guides/online-payments/checkout-api/v2/other-payment-ways.pt.md [hunk 5]
```diff
@@ -2242,7 +2246,8 @@ Consulte os meios de pagamento disponíveis sempre que necessite.
...
```
Comment (elvisjhonataoliveira) [inline guides/online-payments/checkout-api/v2/other-payment-ways.pt.md:2249]:
```suggestion
var response = await mercadopago.get("/v1/payment_methods");
```

File: guides/online-payments/checkout-api/v2/other-payment-ways.pt.md [hunk 6]
```diff
@@ -2892,7 +2897,8 @@ Consulte os meios de pagamento disponíveis sempre que necessite.
...
```
Comment (elvisjhonataoliveira) [inline guides/online-payments/checkout-api/v2/other-payment-ways.pt.md:2900]:
```suggestion
var response = await mercadopago.get("/v1/payment_methods");
```

File: guides/online-payments/checkout-api/v2/other-payment-ways.pt.md [hunk 7]
```diff
@@ -3335,7 +3341,8 @@ Consulte os meios de pagamento disponíveis sempre que necessite.
...
```
Comment (elvisjhonataoliveira) [inline guides/online-payments/checkout-api/v2/other-payment-ways.pt.md:3344]:
```suggestion
var response = await mercadopago.get("/v1/payment_methods");
```

**Review by elvisjhonataoliveira — CHANGES_REQUESTED** [review]
Acho que precisa adicionar o `var` antes de declarar a variável.

**Comment by lucmantovani** [comment]
> Acho que precisa adicionar o `var` antes de declarar a variável.

Ajustado!

**Review by nicolassalvanes — APPROVED** [review]

**Pull request merged**